### ISSUE TITLE
Docker build fails on a fresh pull

### ISSUE THREAD

**Issue by rwblair** [ccnmtl/rolf#94]
Error received:
```
Step 2 : ADD wheelhouse /wheelhouse
stat wheelhouse: no such file or directory
```

**Comment by thraxil** [comment]
Sorry, I haven't really gotten around to documenting the docker stuff yet. It's a two-step build process (so the final docker image can be much smaller and not have the entire compiler/dev toolchain installed). The Makefile has some more details:

https://github.com/ccnmtl/rolf/blob/master/Makefile#L72

And the base and builder images it is using are coming from here: https://github.com/ccnmtl/docker/

The short version is: instead of running `docker build .`, just do `make build`.
